lib/appium-for-mac.js

```diff
@@ -55,7 +59,7 @@ class AppiumForMac {
                   `signal ${signal}`;
         log.error(msg);
       });
-      log.info(`Spawning AppiumForMac with: ${this.appium4macdriver}`);
```

Comment:
Anyone did not use `this. appium4macdriver`. So, the value was always undefined.

Reply:
Why where we spawning?

Reply:
Handled the process by https://github.com/appium/node-teen_process/blob/56a139cbf86b819398d7736a442b6c4a985bc716/lib/subprocess.js#L23

The process will stop in delete the session in general.
https://github.com/appium/appium-mac-driver/blob/master/lib/driver.js#L58
So, the kill all process aims to make a new process run safer.